server/service/src/sync/translation_and_integration.rs

```diff
@@ -134,7 +134,11 @@ impl<'a> TranslationAndIntegration<'a> {
 }
 
 impl IntegrationRecords {
-    pub(crate) fn integrate(&self, connection: &StorageConnection) -> Result<(), RepositoryError> {
+    pub(crate) fn integrate(
+        &self,
+        connection: &StorageConnection,
+        is_sync: bool,
```

Comment:
this is needed now, since integration tests use integrate to insert/upsert records that will in the future need to sync, without this toggle, couldn't reuse integrate method in integration test

Reply:
this was removed in favour of updating is_sync_update status (keeps core code 'cleaner')